packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -246,6 +261,20 @@ export class QueryExportState {
         this.editorStore.applicationStore.notifySuccess(
           `Successfully updated query!`,
         );
+
+        LegendQueryTelemetry.logEvent_UpdateQuery(
```

Comment:
@akphi
Right now this gives us created and updated numbers, but this could potential double count queries right. Should we also add a queryable way to do this from the server side to count the number of current saved queries?